### DISCUSSION TITLE
How to save all posts in one folder specified ?

### DISCUSSION THREAD

**Discussion by SamboMambo** [Serene-Arc/bulk-downloader-for-reddit#936] [Q&A]
so i run the command for a -user, --submitted, and it saves all the posts but puts them in folder named after each subreddit.

How can  i have it save all posts in the specified folder instead of each folder for each subreddit ?

**Comment by Serene-Arc** [comment]
Please read the section of the README on folder and filename schemes. That will give you what you'd like.
